packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -147,7 +147,7 @@ async function copyJsModule(from, to) {
     await copy(from, to, {
         filter: async path =>
             (await promises.lstat(path)).isDirectory()
-                ? !/android|ios|windows|macos|.?(github|gradle)|__(tests|mocks)__|docs|jest|examples?/.test(
+                ? !/^(android|ios|windows|macos|.github|.gradle|__(tests|mocks)__|docs|jest|examples?)$/.test(
```

Comment:
`.` should be escaped

Reply:
Done

Reply:
`?` is not an escape

Reply:
<img width="867" alt="image" src="https://user-images.githubusercontent.com/45102481/105832134-dadd5100-5fc7-11eb-95f7-94a070c7e25f.png">

Reply:
Fixed